src/wms/src/main/java/org/geoserver/wms/WMSInfoImpl.java

```diff
@@ -84,6 +84,8 @@ public class WMSInfoImpl extends ServiceInfoImpl implements WMSInfo {
 
     private Boolean defaultGroupStyleEnabled;
 
+    private boolean autoEscapeTemplateValues;
```

Comment:
I was about to ask "what about equals/hashcode/toString"?
But it seems none of them has been implemented in WMSInfoImpl... oooh well, carry on...